crates/tx5-connection/src/conn.rs

```diff
@@ -367,6 +367,7 @@ async fn webrtc_task(
             break;
         }
     }
+    netaudit!(DEBUG, a = "webrtc task closed, sending WebrtcClosed",);
```

Comment:
When connections do close, we still want to know why and currently don't have any information to help trace where the disconnection starts. I've added a bit of extra tracing where I think we're missing information.